submissions/tosinKolawole/html-forms/contact-form.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contact Form</title>
+</head>
+<body>
+    <h1>LYDIA STORES</h1>
+    <div>
+        <ul>
+           <li><a href="file:///C:/Users/TOSIN/Desktop/Dufuna/index%20product%20pg.html">Product</a></li> 
+           <li><a href="file:///C:/Users/TOSIN/Desktop/Dufuna/html-forms/html-forms.html">Contact us</a></li><br>
```

Comment:
The directories indicated in the links are absolute and will only work at your end because I don't have any 'Users' or 'TOSIN' folder at my end. You can simply use '..html-forms/product-page.html' for the product page and 'contact-form.html' for contact.